scripts/ci/travis.sh

```diff
@@ -54,6 +54,11 @@ targets="cardano-sl cardano-sl-lwallet cardano-sl-tools"
 #  to_build="$to_build cardano-sl-$prj"
 #done
 
+# install cpphs, because Stack can't figure out by itself that it
+# should be installed
+stack --nix --no-terminal --local-bin-path daedalus/ install cpphs \
```

Comment:
@domenkozar like this?

![image](https://user-images.githubusercontent.com/1523306/28743607-4878dfce-7456-11e7-8613-f8051bf5cc87.png)